dags/analyzer_helper/discourse/transform_raw_info.py

```diff
@@ -0,0 +1,93 @@
+from analyzer_helper.discourse.utils.convert_date_time_formats import DateTimeFormatConverter
+
+class TransformRawInfo:
+
+    def __init__(self):
+        self.converter = DateTimeFormatConverter()
+    
+    def create_data_entry(self, raw_data: list, interaction_type: str = None, interaction_user: int = None):
+        metadata = {
+            "channel_id": raw_data.get("category_id"),
+            "thread_id": raw_data.get("topic_id"),
+            "bot_activity": False,
+        }
+
+        if interaction_type == "reaction":
+            return {
+                "actions": [],
+                "author_id": interaction_user,
+                "date": self.converter.convert_to_datetime(raw_data.get("created_at")),
+                "interactions": [
+                    {
+                        "name": "reaction",
+                        "type": "emitter",  
+                        "users_engaged_id": [str(raw_data["post_id"])],
```

Comment:
the `post_id` is not a user here. Please make sure we're assigning the author to be the users_engaged